modules/openshift-argocd-concept.adoc

```diff
@@ -0,0 +1,12 @@
+[id="openshift-argocd-concept"]
+= OpenShift ArgoCD
+include::modules/common-attributes.adoc[]
+include::modules/gitops-document-attributes.adoc[]
+:context: openshift-argocd-concept
+
+:FeatureName: ArgoCD
+include::modules/technology-preview.adoc[leveloffset=+1]
```

Comment:
The entire feature is TP so we do not need to have includes here.
Also we do not add includes to modules, only to assemblies.